app/views/assessments/history.html.erb

```diff
@@ -1,6 +1,6 @@
 <% @title = "Handin History" %>
 <% if @partial then %>
-  <a onclick="hideStudent()">Exit</a>
+  <a class="clickable" onclick="hideStudent()">Exit</a>
```

Comment:
may i know what are these two changes regarding?

Reply:
Currently, when we go to manage submissions and click on a user's email, the popup history panel's "exit" buttons have no href, so hovering over them does not turn the cursor into a pointer.

This change changes that by adding a clickable class, to make it clearer that those are "clickable" links.

Reply:
Nice, thanks for the fix